codechecker_lib/arg_handler.py

```diff
@@ -167,7 +221,7 @@ def handle_server(args):
                                              db_connection_string,
                                              suppress_handler,
                                              args.not_host_only,
-                                             context.db_version_info)
+                                             context)
```

Comment:
Do we really need the whole context object? I try to minimize the usage of it. It is a global object with a lots of information. If not really required I would suggest not to pass it as an argument.